### PR TITLE
Metrics 3.0 snippets

### PR DESCRIPTION
**Pull request by Scooletz**
This PR introduces `NServiceBus.Metrics 3.0` aligned to `v7` of the core.

The prerequisite to merge this is:

- [ ] https://github.com/Particular/NServiceBus.Metrics/pull/65

### CODE REVIEW

**Comment by SimonCropp** [comment]
this will need a rebase

**Comment by Scooletz** [comment]
Done @SimonCropp . Still waiting for the PR to the metrics.

**Comment by SimonCropp** [comment]
given metrics v3 has breaking changes, why is the upgrade guide not in this PR?

**Comment by Scooletz** [comment]
@SimonCropp As far as I see, all the methods that are removed were already described in this guide: https://docs.particular.net/nservicebus/upgrades/metrics-1to2 with the mention of them being obsoleted. 
Now, should I repeat 2-3 migration guide with the same content? Should I create a short upgrade note? Or maybe I should just leave it as it is. I don't know what's the best choice and happy to follow good doco recommendations.

**Comment by SimonCropp** [comment]
@Scooletz sorry u r correct. ignore my comment

**Comment by Scooletz** [comment]
No worries @SimonCropp Just wanted to clarify.

**Comment by Scooletz** [comment]
Ready for a review @Particular/docs-maintainers

File: Snippets/Metrics/Metrics_3/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿namespace Metrics_1
```
Comment (weralabaj) [inline Snippets/Metrics/Metrics_3/Configuration.cs]: we usually skip namespaces in snippets

Comment (weralabaj) [reply Snippets/Metrics/Metrics_3/Configuration.cs]: once removed just merge it ;)

**Comment by Scooletz** [comment]
Removed all namespaces @weralabaj

**Pull request merged**